submissions/emmanuelEsuong/introductionToHtml/index.html

```diff
@@ -0,0 +1,214 @@
+<<!DOCTYPE html>
+<html lang="en">
+<head>
+    <p> <img src="Image/Foodco upmata.PNG" controls></img></p>
```

Comment:
You can't add this elements in the head element.

Reply:
Thanks for the reviews. 

Please is it the paragraph (I. e  <p) or the image itself ?

Reply:
Both